src/App.js

```diff
@@ -28,38 +23,31 @@ import './App.css';
 const localizeProperties = (fn) => React.createElement(injectIntl(({ intl }) => fn(intl)));
 
 type AppContext = {
-    bluetoothApiIsAvailable: boolean,
-    speechRecognitionApiIsAvailable: boolean
+    bluetoothApiIsAvailable: boolean
 };
 
 type AppSettings = {
-    dashSupport: boolean,
-    editorMode: EditorMode,
-    language: string
+    language: string,
+    dashSupport: boolean
```

Comment:
Let's remove the `dashSupport` setting and always show the "Connect to Dash" control.

https://issues.fluidproject.org/browse/C2LC-91 will make the control disabled when Bluetooth isn't available